tools/connect_start_end_points.py

```diff
@@ -91,18 +92,18 @@ def get_points_on_line(line_col, copy_fields=list(),
                 rest_lengte_offset = 0
             
             nr = nr_full
-            
+            # TODO: change to maximum representative length
             # indien extra punt nodig is op de restlengte
-            if use_rest is True:
-                
+            if max_repr_length is True:
```

Comment:
snap ik niet, bij de parameters is het '=150'
code technisch; 'isTrue' kan je weglaten

Reply:
Ik denk dat dit ergens tussen verschillende commits is. In de uiteindelijke code staat dit helemaal niet meer. Ik heb 'use_rest' gerefactored tot 'max_repr_length', dus ten eerste komt dat te staan als in line 97. Aangezien dat er in de oude versie ook al stond. Wel vreemd dat dat dan te zien is nog.